vpr/src/draw/intra_logic_block.cpp

```diff
@@ -547,7 +547,8 @@ void draw_logical_connections(ezgl::renderer* g) {
 
     g->set_line_dash(ezgl::line_dash::none);
 
-    constexpr float NET_ALPHA = 0.0275;
+    //constexpr float NET_ALPHA = 0.0275;
+    float NET_ALPHA = get_net_alpha();
```

Comment:
It seems like alpha only controls the drawn net alpha when 'Toggle Nets:' is set to 'Logical Connections'.

We should also update drawnets (responsible for drawing nets with 'Toggle Nets:' set to 'Nets, I think?) also uses this alpha value.